### PR TITLE
Ins 3081 new logicrunner tests

### PR DESCRIPTION
**Pull request by Tsovak**
**- What I did**
new logicrunner tests
**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: functest/test-data/simple_contract_with_sleep.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package main
```
Comment (LinnikD) [inline functest/test-data/simple_contract_with_sleep.txt]: why not to move all contracts to separate files?

Comment (ruz) [reply functest/test-data/simple_contract_with_sleep.txt]: what? cuz code of contract crucial to the test and it's way harder to understand test when code is further away.

Comment (ruz) [reply functest/test-data/simple_contract_with_sleep.txt]: Please don't do this.

Comment (ruz) [reply functest/test-data/simple_contract_with_sleep.txt]: keep style of the tests intact and follow it. if you want to introduce functest/test-data/ directory then do it in a separate branch and for all tests.

**Pull request merged**